docker-compose-local.yml

```diff
@@ -4,6 +4,71 @@ volumes:
   dynamodb_data:
 
 services:
+  admin:
+    command: [ "-c", "trap : INT TERM QUIT; sleep infinity & wait" ]
+    entrypoint: /bin/bash
+    image: coinbase/chainstorage
+    environment:
+      - AWS_ENDPOINT_OVERRIDE=http://localstack:4566
+      - CADENCE_ENDPOINT_OVERRIDE=temporal:7233
+    restart: always
+    depends_on:
+      - localstack
+      - temporal
+      - api
+  cron:
+    command: [ "./cron" ]
+    image: coinbase/chainstorage
+    environment:
+      - AWS_ENDPOINT_OVERRIDE=http://localstack:4566
+      - CADENCE_ENDPOINT_OVERRIDE=temporal:7233
+      - SDK_ENDPOINT_OVERRIDE=api:9090
+# Only use CLIENT_CONFIG_OVERRIDE for testing purposes if the config files could not be set up
```

Comment:
Wondering when do customers need to set up CLIENT_CONFIG_OVERRIDE? isn't the override already supported by config file?

Reply:
This is to support the use case when developers wants to publish the ChainStorage docker to a repo, say docker-hub, for either internal or external use, and when another developer wants to use Chainsforemr, it can simply run docker compose and override the client configs, because they cannot modify the config files in ChainStorage.